cachito/workers/pkg_managers/golang.py

```diff
@@ -84,6 +91,7 @@ def resolve_gomod_deps(app_source_path, request_id, dep_replacements=None):
             parts = [part for part in line.split(' ') if part not in ('', '<nil>')]
             if len(parts) == 1:
                 # This is the application itself, not a dependency
+                module_name = parts[0]
```

Comment:
Can we add a sanity check if `module_name` is not `None`? We should raise an error as that is unsupported/unexpected behavior.